packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1868,6 +1898,14 @@ describe('TransactionController', () => {
       expect(controller.state.transactions[0]?.txReceipt?.gasUsed).toBe(
         externalTransactionReceipt.gasUsed,
       );
+      expect(controller.state.transactions[0]?.history).toBeDefined();
```

Comment:
As this is a special case where the snapshot can be generated, can we assert this in a separate test?

Reply:
This is fixed in https://github.com/MetaMask/core/commit/7a071b74b1a9eb7d37de6115cd141189bbc8911b